app/views/holds/_holds_ready_table.html.erb

```diff
@@ -15,12 +15,12 @@
     <tbody>
     <% @holds_ready.each do |hold| %>
       <tr>
-        <td>
+        <td scope="row">
           <%= render SelectAllChildComponent, key: hold.key, target_keyword: 'ready', name: 'hold_list[]' %>
         </td>
-        <th scope="row">
```

Comment:
Ok, this does make sense. Initially I'd coneived of the item summary being a kinda of table header (like a ["Table with two headers"](https://www.w3.org/WAI/tutorials/tables/two-headers/)) but it isn't 👍